README.md

```diff
@@ -23,6 +23,7 @@ kubectl create -f manifests/xgboost-operator/base/cluster-role.yaml
 kubectl create -f manifests/xgboost-operator/base/crd.yaml
 kubectl create -f manifests/xgboost-operator/base/service-account.yaml
 kubectl create -f config/rbac/rbac_role_binding.yaml
+kubectl create -f manifests/xgboost-operator/base/deployment.yaml
```

Comment:
Can we just apply -f on `manifests/xgboost-operator/base`?

Reply:
Thank you, I think that's a very good idea, I will try to test it and see how it work.

Reply:
I tried, but it doesn't work.
Seems like we need a namespace file.
And there are version unmatched problem for kustomization and params file.
The following are the error I faced:
 ClusterRoleBinding.rbac.authorization.k8s.io "xgboost-operator" is invalid:subjects[0].namespace: Required value
no matches for kind "Kustomization" in version "kustomize.config.k8s.io/v1beta1"
 error validating "manifests\\xgboost-operator\\base\\params.yaml": error validating data: [apiVersion not set, kind not set]

Reply:
I think the easy way to do this is to create a integrated yaml file just what mpi-operator did